docs/code-samples/settlement/daml.yaml

```diff
@@ -19,15 +19,15 @@ data-dependencies:
   - .lib/daml-finance-interface-instrument-base.dar
   - .lib/daml-finance-interface-instrument-token.dar
   # - .lib/daml-finance-interface-lifecycle.dar
```

Comment:
can we move commented-out deps further below?

Reply:
done